docs/user-guide/deployments.md

```diff
@@ -221,6 +221,10 @@ template is different than `.spec.template` or if the total number of such pods
 exceeds `.spec.replicas`. It will bring up new pods with `.spec.template` if
 number of pods are less than the desired number.
 
+### History
+
+`.spec.revisionHistoryLimit` is an optional field that specifies the number of old replicasets to keep around for rollback purposes. If not specified, the history of every single rollout ever will be preserved, which is probably not what you want.
```

Comment:
I made the first change, but I'm not entirely sure about the second. There is a cost in preserving in etcd the history of all replicasets ever and I think that should be made more obvious.